### PR TITLE
[fix](stats) Ignore complex type by default when collect column statistics

### PR DESCRIPTION
**Pull request by Kikyou1997**
## Proposed changes

By default, if it contains any complex type in Analyze stmt submitted by user and error would be thrown before this PR.

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by Kikyou1997** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/qe/SessionVariable.java [hunk 1]
```diff
@@ -1028,7 +1028,7 @@ public void setMaxJoinNumBushyTree(int maxJoinNumBushyTree) {
     public boolean enableCTEMaterialize = true;
 
     @VariableMgr.VarAttr(name = IGNORE_COMPLEX_TYPE_COLUMN)
-    public boolean ignoreColumnWithComplexType = false;
+    public boolean ignoreColumnWithComplexType = true;
```
Comment (morrySnow) [inline fe/fe-core/src/main/java/org/apache/doris/qe/SessionVariable.java:1031]: change the session variable name to `enable_analyze_complex_type_column` and set default value to `false`

**Comment by Kikyou1997** [comment]
run buildall

**Comment by hello-stephen** [comment]
(From new machine)TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 45.62 seconds
 stream load tsv:          507 seconds loaded 74807831229 Bytes, about 140 MB/s
 stream load json:         20 seconds loaded 2358488459 Bytes, about 112 MB/s
 stream load orc:          64 seconds loaded 1101869774 Bytes, about 16 MB/s
 stream load parquet:          30 seconds loaded 861443392 Bytes, about 27 MB/s
 insert into select:          29.0 seconds inserted 10000000 Rows, about 344K ops/s
 storage size: 17161093188 Bytes

File: fe/fe-core/src/main/java/org/apache/doris/qe/SessionVariable.java [hunk 2]
```diff
@@ -349,7 +349,7 @@ public class SessionVariable implements Serializable, Writable {
 
     public static final String ENABLE_SCAN_RUN_SERIAL = "enable_scan_node_run_serial";
 
-    public static final String IGNORE_COMPLEX_TYPE_COLUMN = "ignore_column_with_complex_type";
+    public static final String IGNORE_COMPLEX_TYPE_COLUMN = "enable_analyze_complex_type_column";
```
Comment (englefly) [inline fe/fe-core/src/main/java/org/apache/doris/qe/SessionVariable.java:352]: why make the name and content mismatch?

**Review by englefly — DISMISSED** [review]

**Review by morrySnow — DISMISSED** [review]

**Comment by Kikyou1997** [comment]
run buildall

**Review by morrySnow — APPROVED** [review]

**Review by englefly — APPROVED** [review]

**Pull request merged**